src/ucl_util.c

```diff
@@ -1919,7 +1919,7 @@ ucl_inherit_handler (const unsigned char *data, size_t len,
 
 	/* Some sanity checks */
 	if (parent == NULL || ucl_object_type (parent) != UCL_OBJECT) {
-		ucl_create_err (&parser->err, "Unable to find inherited object %*.s",
+		ucl_create_err (&parser->err, "Unable to find inherited object %.*s",
```

Comment:
Oh, I always forget about the order of `*` and `.`. In Rspamd, I have the own printf implementation which has distinction between `%.*s` and `%*.s`. But here we use a generic libc `printf`...